pkg/ebpf/c/common/consts.h

```diff
@@ -24,8 +24,12 @@
 #define MAX_STR_ARR_ELEM      38 // TODO: turn this into global variables set w/ libbpfgo
 #define MAX_ARGS_STR_ARR_ELEM 15
 #define MAX_PATH_PREF_SIZE    64
+#define MAX_DATA_FILTER_STR_SIZE 128
```

Comment:
Can you please remind me why this can't be 256 bytes long?

Reply:
IIRC perf tests were done and it was decided to use 128.

Reply:
We initially used `event id`, and the next available mask was 128 (for reverse string). However, now we're using 256 bytes since event id has been removed from the key of the inner map.